client/webserver/site/src/js/register.js

```diff
@@ -34,19 +34,17 @@ export default class RegistrationPage extends BasePage {
     body.querySelectorAll('.form-closer').forEach(el => Doc.hide(el))
 
     // SET APP PASSWORD
-    bindForm(page.appPWForm, page.appPWSubmit, () => { this.setAppPass() })
+    bindForm(page.appPWForm, page.appPWSubmit, () => this.setAppPass())
 
     // NEW DCR WALLET
     // This form is only shown if there is no DCR wallet yet.
-    this.walletForm = new NewWalletForm(app, page.newWalletForm, () => {
-      this.changeForm(page.newWalletForm, page.dexAddrForm)
-    })
+    this.walletForm = new NewWalletForm(app, page.newWalletForm, () =>
+      this.changeForm(page.newWalletForm, page.dexAddrForm))
```

Comment:
Please don't change this. I really prefer not to use an un-bracketed two-liner syntax. For consistency, if a block is more than one line, it's at least three.